docs/administration/upgrading/guide/automate-upgrades.md

```diff
@@ -0,0 +1,284 @@
+
+$rawRequest = @{
+  Id = "maintenance";
+  IsInMaintenanceMode = $IsMaintenanceMode;  ## Change this to false when you want to take out of maintenance mode
+  Links = @{
+    Self = "/api/maintenanceconfiguration";
+  }
+}
+$jsonRequest = $rawRequest | ConvertTo-Json
+
+Write-Host "Sending in the request $jsonRequest"
+
+$maintenanceUrl = "$OctopusUrl/api/maintenanceconfiguration"
+Write-Host "Setting maintenance mode $maintenanceUrl"
+$maintenanceResponse = Invoke-RestMethod $maintenanceUrl -Headers $header -Method POST -Body $jsonRequest
+
+Write-Host "Maintenance's response: $maintenanceResponse"
+```
+
+### Stop the instance
+
+[Octopus.Server.exe](/docs/octopus-rest-api/octopus.server.exe-command-line/index.md) is the command line interface, or CLI, of the Octopus Manager.  The below script will stop the service.  Before it does that it will drain the node and wait for all the tasks to complete before stopping the service.
+
+```PowerShell
+Set-Location "${env:ProgramFiles}\Octopus Deploy\Octopus" 
+
+& .\octopus.server.exe node --instance="OctopusServer" --drain=true --wait=0
+& .\octopus.server.exe service --instance="OctopusServer" --stop
+```
+### Backup the SQL Server database
+
+The simplest backup possible is a full database backup.  Execute the below T-SQL command to save a backup to a NAS or file share.
+
+```
+BACKUP DATABASE [OctopusDeploy]
+          TO DISK = '\\SomeServer\SomeDrive\OctopusDeploy.bak'
+             WITH FORMAT;
+```
+
+The `BACKUP DATABASE` T-SQL command has dozens of various options.  Please refer to [Microsoft's Documentation](https://docs.microsoft.com/en-us/sql/relational-databases/backup-restore/create-a-full-database-backup-sql-server?view=sql-server-ver15) or consult a DBA as to which options you should use.
+
+### Installing the MSI
+
+The Octopus Deploy MSI installation can be automated by calling `msiexec.exe` directly or using a third-party tool such as Chocolatey.
+
+:::warning
+Please run `msiexec.exe` as an administrator.  It is performing an installation, and you will be prompted by Windows to confirm.
+:::
+
+#### Using msiexec.exe
+
+Windows includes [msiexec.exe](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/msiexec) as part of the base installation.
+
+The `msiexec.exe` can be invoked via PowerShell by using the Start-Process command.  This example will send in the `/i` and `/quiet` arguments to install the MSI without user interaction.
+
+```PowerShell
+$msi = "C:\Temp\OctopusDeploy.2020.4.2.msi"
+Write-Output "Installing MSI $msi" 
+$msiExitCode = (Start-Process -FilePath "msiexec.exe" -ArgumentList "/i $msi /quiet" -Wait -Passthru).ExitCode 
+Write-Output "Server MSI installer returned exit code $msiExitCode" 
+```
+
+#### Using Chocolatey
+
+[Chocolatey](https://chocolatey.org) is a third-party package management tool designed to work with Windows.  If you are coming from Linux, it is similar to apt or yum.  
```

Comment:
Suggestion only. I'm not sure this adds anything as apt and yum are generally known as package managers.
```suggestion
[Chocolatey](https://chocolatey.org) is a third-party package management tool designed to work with Windows.
```